MANIFEST.in

```diff
@@ -1,6 +1,7 @@
 include *.txt
 recursive-include wof/flask/templates *
 recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples/production_configs *
```

Comment:
@lsetiawan and I are confused here. Not about the change you're making now (including production_configs), but about why this line was needed to begin with:
```
recursive-include wof/examples/flask/odm2/timeseries *
```

We're assuming that all module folders under `wof/examples/flask/` would be included, unless they're excluded explicitly via `prune`. @lsetiawan just confirmed this expectation.

SO: our recommendation would be to remove that odm2/timeseries line. It's not doing anything wrong, but it's not needed and that kind of causes confusion (to us at least!)